list_grabber.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+#
+# Abbas Razaghpanah (arazaghpanah@cs.stonybrook.edu)
+# January 2015, Stony Brook University
+#
+# list_grabber.py: a script to hit a given URL and download
+# all of the csv files linked on the index page.
+# This can be run periodically to update URL lists for different 
+# countries, censorship products, etc. using a URL repository.
+# The script assumes HTTP basic authentication for the repository.
+#
+# For CSV files that are of form XX.csv, it is assumed that it
+# belongs to a country XX and it will be copied to the related
+# directory (which is created if it doesn't exist). Everything
+# else will go to "global".
+
+import argparse
+import os
+from os import path
+import re
+import requests
+from requests.auth import HTTPDigestAuth
+import string
+from urlparse import urljoin
+
+def parse_args():
```

Comment:
PEP8 calls for two new lines between functions and between imports and anything else